platform/jewel/ide-laf-bridge/src/main/kotlin/org/jetbrains/jewel/bridge/theme/SwingBridgeTheme.kt

```diff
@@ -35,6 +39,8 @@ public fun SwingBridgeTheme(content: @Composable () -> Unit) {
             LocalNewUiChecker provides BridgeNewUiChecker,
             LocalDensity provides scaleDensityWithIdeScale(LocalDensity.current),
             LocalClipboard provides remember { JewelBridgeClipboard() },
+            LocalUriHandler provides BridgeUriHandler(),
+            LocalLabelProvider provides BridgeLabelProvider(),
```

Comment:
Nit: IMO we should movi this to the `JewelComposePanel` functions. Some of them are without theme, but we still want to handle the links properly in the IDE